src/edge/views.py

```diff
@@ -550,6 +554,29 @@ def parse_arguments(self, request):
 class GenomeRecombinationView(GenomeOperationViewBase):
     DEFAULT_HA_LENGTH = 30
 
+    @staticmethod
+    def validate_annotations(cassette, annotations):
+        """
+        If user supplied annotations for the donor sequence, to make sure we
+        can precisely use the annotation and not leave any ambiguity, we
+        require the donor dna to not contain overhangs and backbone
+        modifications. This method returns True if that's the case, and
+        required fields for annotations exist.
+        """
+
+        if re.match(r'^[A-Za-z]+$', cassette) and \
```

Comment:
how does re.match(r'^[A-Za-z]+$', cassette) check the donor not have overhangs?

Reply:
LIMS sequence that has overhang or backbone modifications would include characters like / and ( and )

Reply:
Got it!